pkg/snapstore/swift_snapstore_test.go

```diff
@@ -150,3 +178,50 @@ func handleDeleteObject(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusNotFound)
 	}
 }
+
+// handleBulkDeleteObject creates an HTTP handler at `/testContainer/testObject` on the test handler mux that
+// responds with a `Delete` response.
+func handleBulkDeleteObject(w http.ResponseWriter, r *http.Request) {
+	objectMapMutex.Lock()
+	defer objectMapMutex.Unlock()
+
+	var jsonResponse bulkDeleteJSONResponse
+
+	buf := new(bytes.Buffer)
+	if _, err := io.Copy(buf, r.Body); err != nil {
+		errorMessage := fmt.Sprintf("failed to read content %v", err)
+		logrus.Errorf(errorMessage)
+		jsonResponse.ErrorStrings = append(jsonResponse.ErrorStrings, errorMessage)
+		w.WriteHeader(http.StatusOK)
+		return
```

Comment:
The Bulk Delete Response [docs](https://docs.openstack.org/swift/latest/api/bulk-delete.html#bulk-delete-response):

> When Object Storage processes the request, it performs multiple sub-operations. Even if all sub-operations fail, the operation returns a 200 status. The bulk operation returns a response body that contains details that indicate which sub-operations have succeeded and failed. Some sub-operations might succeed while others fail. Examine the response body to determine the results of each delete sub-operation.

Since the response from Swift would be the 200 code always, I've emulated that in the tests too, and information regarding failure and success regarding the sub-operations is sent in the body of the response as done here.
I've not emulated the response body content structure of Swift since it would be unnecessary in unit tests.